apps/journeys-admin/src/components/Team/TeamManageDialog/TeamManageWrapper/TeamManageWrapper.tsx

```diff
@@ -0,0 +1,55 @@
+import { ReactElement, ReactNode } from 'react'
+import { UserTeamList } from '../UserTeamList'
+import { UserTeamInvitesList } from '../UserTeamInvitesList'
+import { useTeam } from '../../TeamProvider'
+import { useCurrentUser } from '../../../../libs/useCurrentUser'
+import { useUserTeamsAndInvitesQuery } from '../../../../libs/useUserTeamsAndInvitesQuery'
+import { UserTeamInviteForm } from '../../UserTeamInviteForm'
+import { GetUserTeamsAndInvites } from '../../../../../__generated__/GetUserTeamsAndInvites'
+
+interface TeamManageWrapperProps {
+  children: (props: {
+    data?: GetUserTeamsAndInvites
+    UserTeamList: ReactElement
+    UserTeamInviteList: ReactElement
+    UserTeamInviteForm: ReactElement
```

Comment:
apologies. since these are now no longer components can we camelcase them?
```suggestion
    userTeamList: ReactElement
    userTeamInviteList: ReactElement
    userTeamInviteForm: ReactElement
```